hack/common.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+get_image_tag() {
+    local current_commit today
+    current_commit="$(git rev-parse HEAD)"
+    today="$(date +%Y%m%d)"
+    echo "v${today}-${current_commit:0:7}"
+}
+
+SSP_DIR="$(readlink -f $(dirname $0)/../)"
+WORK_DIR="/go/src/github.com/kubevirt/ssp-operator"
+REGISTRY=${REGISTRY:-quay.io/kubevirt}
```

Comment:
Can you please change this to `TEST_IMAGE_REGISTRY` and `TEST_IMAGE_REPOSITORY`?

Reply:
Changed